tests/lsp_features/completion.zig

```diff
@@ -1672,6 +1672,24 @@ test "completion - enum completion on out of bound parameter index" {
     , &.{});
 }
 
+test "completion - delegated declaration" {
```

Comment:
Could you move this test case to some of the other tests that deal with functions like `completion - function call`